src/04-modality-specific-files/05-task-events.md

```diff
@@ -1,48 +1,47 @@
 # Task events
 
+The purpose of this file is to describe timing and other properties of events
+recorded during a run.
+Events MAY be either stimuli presented to the participant or participant responses.
+A single event file MAY include any combination of stimuli and response events.
+Events MAY overlap in time.
+Please mind that this does not imply that only so called "event related" study designs
+are supported (in contrast to "block" designs) - each "block of events" can be
+represented by an individual row in the `_events.tsv` file (with a long
+duration).
+
 Template:
 
 ```Text
 sub-<label>/[ses-<label>]
-    func/
+    <data_type>/
```

Comment:
Events do not only concern `func` (at least mentioned in `beh`) so I suggest making that more general.

**On that note**

There are several bids-examples datasets for "eeg", "meg", "ieeg" that have `events.tsv`. But this seems to not be mentioned in the specifications that you can have those files there.

Should we at least mention in the `events` page what data type allow `events.tsv`?

And update the schema accordingly?

Reply:
Forget the part of the schema: I did not realize that the latest version in `master` has the events suffix for `eeg`, `meg`, `ieeg` and that this is rendered automatically in the specification upon build.